Automattic/wp-calypso
# Staging Sites: Fix incorrect check for active plugins array

Related to https://github.com/Automattic/dotcom-forge/issues/2003

## Proposed Changes

In this diff, I propose to fix a condition that checks for active Jetpack plugins. Currently, it doesn't check for array length, so it returns `true` even for an empty array.

## Testing Instructions



1. Create one or more staging sites using the wpcalypso environment
2. Open a `/sites` page 
3. In another browser tab, open any site and then click "Switch site"
4. Compare list of sites in sidebar and Sites page and ensure sites are displayed in the same order
5. Filter both lists by "staging-" and confirm that the same sites are displayed in both places

<img width="322" alt="Screenshot 2023-03-20 at 14 19 18" src="https://user-images.githubusercontent.com/727413/226361195-df8caa03-931b-4737-a304-f2b545186a92.png">

## Pre-merge Checklist



](https://github.com/Automattic/dotcom-forge/issues/2003)
